packages/order-utils/test/signature_utils_test.ts

```diff
@@ -188,12 +182,57 @@ describe('Signature utils', () => {
                 },
             };
 
-            const messagePrefixOpts = {
-                prefixType: MessagePrefixType.EthSign,
-                shouldAddPrefixBeforeCallingEthSign: false,
+            const ecSignature = await ecSignOrderHashAsync(
+                fakeProvider,
+                orderHash,
+                makerAddress,
+                SignerProviderType.EthSign,
+            );
+            expect(ecSignature).to.equal(expectedSignature);
+        });
+        // Note this is due to a bug in Metamask where it does not prefix before signing, this is a known issue and is to be fixed in the future
```

Comment:
Can we add a `// Source: URL` here to the issue or PR, so future dev's can check on it's status?